### PR TITLE
DeepLearning

### PR DESCRIPTION
**Pull request by shengwangXX**

### CODE REVIEW

File: docs/DeepLearning/README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# DeepLearning on QingCloud AppCenter 用户指南
+## 深度学习简介
```
Comment (martinyunify) [inline docs/DeepLearning/README.md]: MD022 Headers should be surrounded by blank lines

File: docs/DeepLearning/README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+- DeepLearning app与QingStor命令行工具集成，[配置](https://docs.qingcloud.com/qingstor/command_line_tools/qsctl.html)
+QingStor相关参数，可以方便的从QingStor拉取数据。如果QingStor选用false，access_key和secret_key则无需填写。
+
+## DeepLearning 测试
+以MNIST数据集为例，分别测试caffe，tensorflow，pytorch。MNIST数据集包含0-9 10个数字，
+训练数据集包含 60,000 个样本，测试数据集包含 10,000 样本，数据集中的每张图片由 28 x 28 个像素点构成。
+> Deep Learning 训练往往需要大量的数据，数据存储经常占用很大的空间。青云QingStor可以存储海量数据，
+用户可以方便的把数据放在QingStor，再使用QingStor命令行工具快速的下载到本地。如果在环境设置中配置好了QingStor(如果没有也可在本地[配置](https://docs.qingcloud.com/qingstor/command_line_tools/qsctl.html))，从QingStor获取数据：  
+```shell
```
Comment (martinyunify) [inline docs/DeepLearning/README.md]: MD031 Fenced code blocks should be surrounded by blank lines

File: docs/DeepLearning/README.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+### 第3步：网络设置
+
+![第3步：网络设置](../../images/DeepLearning/network_config.png)
+
+- 出于安全考虑，所有的集群都需要部署在私有网络中，选择自己创建的已连接路由器的私有网络中。
+
+### 第4步：环境设置
+![第4步：环境设置](../../images/DeepLearning/env_config.png)
+
+- DeepLearning app与QingStor命令行工具集成，[配置](https://docs.qingcloud.com/qingstor/command_line_tools/qsctl.html)
```
Comment (martinyunify) [inline docs/DeepLearning/README.md]: MD032 Lists should be surrounded by blank lines

File: docs/DeepLearning/README.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+2016年AlphaGo战胜李世石，预示我们进入了AI时代。深度学习是AI的核心技术，在图像分类，自然语言处理，无人驾驶等众多领域显示出了强大的能力，各大巨头纷纷投入巨资研发。语音助手，人脸识别，外文翻译等等，AI已融入到了我们生活的方方面面，极大了促进了社会的发展。其中Caffe，TensorFlow，Keras，PyTorch是主流的深度学习框架，拥有强大的社区支持，是实践深度学习不可或缺的工具。  
+### Caffe
+Caffe是一个被广泛使用的深度学习框架，由BVLC开发。Caffe容易上手，训练速度快，组件模块化，并拥有大量训练好的经典模型。Caffe在GPU上训练的性能很好，但只能支持单机多GPU的训练，不支持分布式多机训练。
+### TensorFlow
+TensorFlow由Google大脑主导开发，是一个分布式系统上的大规模深度学习框架。移植性好，可以运行在移动设备上，并支持分布式多机多卡训练，支持多种深度学习模型。TensorFlow还有功能强大的可视化组件TensorBoard，能可视化网络结构和训练过程，对于观察复杂的网络结构和监控长时间、大规模的训练很有帮助。
+### PyTorch
+PyTorch从Torch发展而来，并经过了大量改进，由FaceBook AI团队主导开发。不同于TensorFlow，PyTorch采用动态计算图的方式，并提供良好的python接口，代码简单灵活，使用起来非常方便。内存分配也经过了优化，能支持分布式多机训练。
+### Keras
+Keras是一个高层神经网络API，由Python编写，通过调用其他深度学习框架来进行计算，如Tensorflow、Theano以及CNTK。Keras高度模块化，能快速搭建神经网络，并且非常容易上手。
+# 青云深度学习平台
```
Comment (martinyunify) [inline docs/DeepLearning/README.md]: MD025 Multiple top level headers in the same document

**Review by martinyunify — DISMISSED** [review]

**Comment by martinyunify** [comment]
先把https://travis-ci.org/QingCloudAppcenter/user-guide/builds/296587936?utm_source=github_status&utm_medium=notification 里面的错误改一下吧。

File: docs/DeepLearning/README.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
+# DeepLearning on QingCloud AppCenter 用户指南
```
Comment (rayzhou2017) [inline docs/DeepLearning/README.md]: DeepLearning单词要分开吧？

**Review by rayzhou2017 — APPROVED** [review]

**Review by rayzhou2017 — DISMISSED** [review]

**Review by rayzhou2017 — APPROVED** [review]

**Pull request merged**